Tech-Journal-ConsoleApp/Tech-Journal-ConsoleApp/Program.cs

```diff
@@ -6,7 +6,19 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var journal = new Journal();
+            var sendEmail = new SendEmail();
+            sendEmail.ValidateEmailSettings();
```

Comment:
Is there no 'getEmail' method? It seems odd that after creating objects you immediately validate the settings.